### ISSUE TITLE
Setting `throttle` to 0 does not disable throttling.

### ISSUE THREAD

**Issue by mkrause** [szimek/signature_pad#246]
The docs mention:

> throttle
> (integer) Draw the next point at most once per every x milliseconds. Set it to 0 to turn off throttling. Defaults to 16.

But this doesn't seem to work. Looking at the code I see:

```js
  this.throttle = opts.throttle || 16; // in miliseconds

  if (this.throttle) {
    this._strokeMoveUpdate = throttle(SignaturePad.prototype._strokeUpdate, this.throttle);
  } else {
    this._strokeMoveUpdate = SignaturePad.prototype._strokeUpdate;
  }
```

Since `0` is a falsy value, passing this as an option will happily use the default `16`.

**Comment by szimek** [comment]
@mkrause Uh, my fault. That `|| 16` was my addition to the PR that adds throttling. Would you like to prepare a PR with a fix?

**Comment by mkrause** [comment]
@szimek Done!

**Comment by szimek** [comment]
Thanks!

**Comment by szimek** [comment]
@mkrause Got a question - was there any particular reason why you wanted to disable throttling or was it just a bug you found?

**Comment by mkrause** [comment]
@szimek I encountered a certain timing issue, which would cause an error in the following code:

```js
  // SignaturePad.prototype._strokeUpdate()
  this._data[this._data.length - 1].push({
    x: point.x,
    y: point.y,
    time: point.time,
});
```

This assumes that there is at least one data element already available (`this._data.length - 1`), but in my particular setup this was not always the case. If throttling was on, then between the beginning of the stroke and the update there might be a call to `clear()`.

```js
// Async call to strokeUpdate, leaves the possibility for a clear() between now and the call
this._strokeMoveUpdate = throttle(SignaturePad.prototype._strokeUpdate, this.throttle);
```

Granted, the code I had written was sort of unorthodox. I wrote a controlled React `Signature` component, which would -- every time a `strokeEnd` event occurs -- serialize the updated signature to a data URL, update the internal React state, and then that state change would be fed back to the Signature component causing a complete rerender (`clear()` + `fromDataUrl()`) of the canvas.

If you're interested I could dig up some example code. But I found a workaround that fixes the problem anyway.
